### PR TITLE
Orca MXNetTrainer migration

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/orca/learn/mxnet/estimator.py [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
 import ray.services
 from dmlc_tracker.tracker import get_host_ip
-from zoo.ray.mxnet.mxnet_runner import MXNetRunner
-from zoo.ray.mxnet.utils import find_free_port
+from zoo.orca.learn.mxnet.mxnet_runner import MXNetRunner
+from zoo.orca.learn.mxnet.utils import find_free_port
 
 
-class MXNetTrainer(object):
+class Estimator(object):
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/mxnet/estimator.py:26]: if we still use the `train` API, we can keep it as MXNetTrainer

File: pyzoo/zoo/orca/learn/mxnet/estimator.py [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
     """
-    MXNetTrainer provides an automatic setup for synchronous distributed MXNet training.
+    MXNet Estimator provides an automatic setup for synchronous distributed MXNet training.
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/mxnet/estimator.py:28]: Need to support `DataShards`; see the PR link in https://github.com/analytics-zoo/orca/issues/3

Comment (cyita) [reply pyzoo/zoo/orca/learn/mxnet/estimator.py:28]: ok

File: pyzoo/zoo/orca/learn/mxnet/estimator.py (hunk 1 above)
Comment (cyita) [reply pyzoo/zoo/orca/learn/mxnet/estimator.py]: fixed

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/view/ZOO-PR-PIPELINE/job/ZOO-PR-Validation/3394/

**Pull request merged**